checkov/sast/checks/base_registry.py

```diff
@@ -79,3 +83,13 @@ def _should_skip_check(self, rule: Dict[str, Any]) -> bool:
         if self.runner_filter.should_run_check(check):
             return False
         return True
+
+    def create_temp_rules_file(self) -> None:
+        rules_obj = {'rules': self.rules}
+        with open(self.temp_semgrep_rules_path, 'w') as tempfile:
+            yaml.safe_dump(rules_obj, tempfile)
+        logging.debug(f'created semgrep temporary rules file at: {self.temp_semgrep_rules_path}')
+
+    def delete_temp_rules_file(self) -> None:
+        os.remove(self.temp_semgrep_rules_path)
```

Comment:
I assume this might be called when the file is not created by mistake sometime (from a user or from a bug), so I think you should wrap it in `try/catch FileNotFound`